src/app/pages/sharing/smb/smb-form/smb-form.component.ts

```diff
@@ -107,6 +139,7 @@ export class SmbFormComponent implements OnInit {
     hostsdeny: [[] as string[]],
     home: [false],
     timemachine: [false],
+    timemachine_quota: [],
```

Comment:
This probably results in type being not defined for this field. You should probably have something like:
```
timemachine_quota: [null as number]
```